src/cf-forms/src/molecules/form-fields.less

```diff
@@ -44,11 +44,11 @@
             .lt-ie9 & {
                 height: unit( 20px / @base-font-size-px, em );
                 width: unit( 20px / @base-font-size-px, em );
+                width: auto;
```

Comment:
Hmmm how was this one working? Won't the `width` on 46 get overwritten by `width: auto` in any case?

Reply:
Yes, they were both there already. All I did was manually move `width: auto` up in the ruleset to avoid violating the no duplicates unless they are consecutive rule that I added.